### PR TITLE
Expose ability to enable NNApi in C api

### PR DESCRIPTION
**Pull request by biodi**
Small change that doesn't add any feature, just exposes the existing one.

### CODE REVIEW

**Comment by jdduke** [comment]
I would rather put this in `c_api_experimental.h`. My preference could actually be to expose a pure C wrapper for the NnApiDelegate so that we can use the already-stable api for adding delegates. We will deprecate the `Interpreter::UseNnApi()` method in the future.

**Comment by biodi** [comment]
@jdduke hope that putting in experimental api is enough for now

File: tensorflow/lite/c/c_api_internal.h [hunk 1]
```diff
@@ -49,6 +49,8 @@ struct TfLiteInterpreterOptions {
   void* error_reporter_user_data = nullptr;
 
   std::vector<TfLiteDelegate*> delegates;
+
+  bool useNNAPI = false;
```
Comment (jdduke) [inline tensorflow/lite/c/c_api_internal.h]: use_nnapi

**Review by jdduke — CHANGES_REQUESTED** [review]
One minor nit, otherwise lgtm.

File: tensorflow/lite/c/c_api_internal.h (hunk 1 above)
Comment (biodi) [reply tensorflow/lite/c/c_api_internal.h]: fixed

**Review by jdduke — APPROVED** [review]

**Pull request merged**